worker/src/main/java/gov/cms/ab2d/worker/service/JobProcessingServiceImpl.java

```diff
@@ -115,6 +113,24 @@ public Job processJob(final String jobId) {
         return job;
     }
 
+    private List<Contract> getAttestedContracts(Job job) {
+
+        // when the job is submitted for a specific contract, process the export for only that contract.
+        final Contract jobSpecificContract = job.getContract();
+        if (jobSpecificContract != null && jobSpecificContract.getAttestedOn() != null) {
+            log.info("Job [{}] submitted for a specific attested contract [{}] ", jobSpecificContract.getContractNumber());
+            return Collections.singletonList(jobSpecificContract);
+        }
```

Comment:
What needs to happen if the contract specified in the job is not attested?
Should we return an error indicating the specified contract is not attested and fail the job?

Reply:
Attestation check isn't needed here: it is being done at job creation time.